packages/plugin-user/src/user.js

```diff
@@ -274,15 +274,20 @@ const User = SparkPlugin.extend({
         })
         .catch((err) => Promise.reject(new Error(`failed to set authorization`, err)))
       )
-      .then(() => this.request({
-        service: `atlas`,
-        resource: `users/activations`,
-        method: `POST`,
-        headers,
-        body: options,
-        shouldRefreshAccessToken,
-        requiresClientCredentials
-      }))
+      .then(() => {
+        if (options.spoofUserAgent) {
```

Comment:
how about

```
if (typeof window === `undefined` && ! header[`user-agent`]) {
  header[`user-agent`] = `spark-js-sdk`
}
```

I don't like the idea of masquerading as a different user agent type and I really don't like the idea of being able to opt into it arbitrarily from production code.